### DISCUSSION TITLE
How can I change filament in Orca Slicer 2.2.0?

### DISCUSSION THREAD

**Discussion by bybarbaros** [OrcaSlicer/OrcaSlicer#7866] [General]
Hello friends. There was a filament changing section here to print the part in 2 colors. Now how does this feature work?
![m600x](https://github.com/user-attachments/assets/31dbccd3-cba9-424a-a4e1-9ce2a8819f3e)

**Comment by bybarbaros** [comment]
![resim_2024-12-29_164650314](https://github.com/user-attachments/assets/da289479-4d37-4324-99c0-bef1e0dcfba0)

**Comment by ButtlerDrownPF** [comment]
To change the filament in Orca Slicer 2.2.0, go to the "Filament" tab, add or edit a filament profile, adjust the settings, and save it. For the latest updates, visit the [Slicing Software Download](https://theorcaslicer.com/) page.

**Comment by logikal** [comment]
Do not download Orcaslicer from anywhere but the GitHub releases in this repo. @SoftFever if you wanna delete this spam, please do so.

**Comment by W1SPS** [comment]
I second that:
Do not download Orcaslicer from anywhere but the GitHub releases in this repo. @SoftFever if you wanna delete this spam, please do so.

**Comment by bybarbaros** [comment]
I found the source of the problem friends. The "single extruder multiple filament" option was not active. thank you to friends who support
